Makefile

```diff
@@ -14,7 +14,10 @@ build: ## Build the project
 	section "Cargo build" ;\
 	cargo build
 
-fix:  ## Fix formatting and clippy errors
+fix: ## Fix formatting and clippy errors (deprecated)
+	echo "Use 'make format' instead"
+
```

Comment:
Why not remove the old completely? I mean keeping this has no big benefits I think.

Reply:
My guess is for anyone used to using `make fix` that doesn't see this PR and maybe confused when it's gone. Granted, it's not hard to look at the Makefile but it'll at least tell you it's moved; then maybe just completely remove it at some point

Although I don't have a preference for either way, whether we remove it now or later